conf/ux/expectations.conf.schema.json

```diff
@@ -0,0 +1,220 @@
+{
+    "$schema": "http://json-schema.org/draft/2020-12/schema",
+    "title": "Expectations and Notifications",
+    "description": "Specifies which trips users are expected to label and when they should be reminded to do so",
+    "type": "object",
+    "properties": {
+        "modes": {
+            "description": "The various modes we can be in and their specifications. The active mode is the first mode in this list with a schedule matching the current date or with no schedule.",
+            "type": "array",
+            "items": {
+                "$ref": "#/definitions/modeType"
+            },
+            "contains": {
+                "$comment": "Must contain a \"default\" element -- one with no schedule",
+                "not": {"required": ["schedule"]}
+            }
+        }
```

Comment:
Exactly. The scheduling algorithm iterates through modes from top to bottom and applies the first mode that either has a schedule matching the current date or has no schedule. The schema requires a mode with no schedule so that some mode will always apply.